expression/builtin_math_test.go

```diff
@@ -829,7 +829,7 @@ func TestCos(t *testing.T) {
 		{int64(0), float64(1), false, false},
 		{math.Pi, float64(-1), false, false}, // cos pi equals -1
 		{-math.Pi, float64(-1), false, false},
-		{math.Pi / 2, math.Cos(math.Pi / 2), false, false}, // Pi/2 is some near 0 (6.123233995736766e-17) but not 0. Even in math it is 0.
+		{math.Pi / 2, math.Cos(math.Pi / 2), false, false}, // Pi/2 is some near 0 (6.123233995736766e-17) but not 0. Even in mathutil it is 0.
```

Comment:
Please revert, I think the comment is still correct.

Reply:
Done